project/frontend/src/containers/Explanation/Explanation.js

```diff
@@ -1,45 +1,60 @@
-          kielbasa t-bone fatback boudin. Rump sirloin pork chop jowl ribeye.
-          Hamburger brisket capicola chicken beef ham hock pork chop jerky
-          ribeye rump tail pig. Pastrami venison pork tongue, burgdoggen
-          capicola t-bone chuck salami ribeye corned beef pig drumstick jowl.
-          Flank buffalo ham hock doner filet mignon sausage. Doner tongue
-          pancetta flank. Shank pork beef ribs buffalo, rump jowl salami brisket
-          spare ribs pork loin short ribs ham hock.
-        </p>
-        <p>
-          Ham hock porchetta pork, meatloaf ball tip bacon buffalo. Salami
-          fatback rump venison porchetta, ham hock tenderloin chuck strip steak
-          turducken short ribs turkey chicken pork chop. Biltong pork loin chuck
-          meatball, bresaola brisket pork belly short loin turkey alcatra kevin
-          doner drumstick corned beef short ribs. Tenderloin kielbasa shoulder
-          hamburger buffalo strip steak. Sausage shank hamburger t-bone pork
-          belly pork jerky porchetta jowl.
+          SGonks work along similar ideas to virtual trading platforms, but
+          instead of investing in stocks, the player can invest “Trend Bucks” in
+          a Google search query. By buying into a search at a given point in
+          time, they can cash out later and earn or lose “Trend Bucks" based on
+          increased or decreased global interest in that query, as indicated by
+          Google trends data. This provides an interactive way for teams to
+          compete with one another to predict search trends overtime.
         </p>
       </div>
-      <Link to="/" className={classes.BackButton}>Back</Link>
+
+      <div className={classes.ExplanationContainer}>
+        <h1>How does it work?</h1>
+        <ol>
+          <li>
+            You can find all the competition you’re in in the or create a
+            competition if you don’t have any.
+          </li>
+          <li>
+            The aim of the game is to invest in the Google Search that you think
+            it’s popularity is going to increase in the future and end up with
+            the most trend bucks at the end of the competition.
+          </li>
+          <li>
+            You will receive trends bucks based on the increased percentage of
+            your google search investments. Same thing applies to losing trend
+            bucks when the popularity of your investment decreases.
+          </li>
+          <li>
+            You can find all your current information in the game at my sGonks
+            page. This includes your current trend buck, your initial worth,
+            your google search investments and it’s current value.
+          </li>
+          <li>
+            You can find Google searches to invest in at sGonks Market page.
+          </li>
+          <li>
+            To see your ranking in your team and information about your
```

Comment:
To see your ranking and information about the performance of other competitors, go to the Competition page.